src/app/routes/_app/rulesets/$rulesetSlug/rulebooks/$rulebookSlug/edit/-rulebookPageDetailsEdit.module.css

```
@@ -0,0 +1,162 @@
+  border-top: 1px solid var(--mantine-color-default-border);
+}
+
+.region {
+  min-width: 0;
+  padding-block: var(--mantine-spacing-md);
+  border-bottom: 1px solid var(--mantine-color-default-border);
+  transition: opacity var(--transition-fast);
+}
+
+.region[data-active] {
+  box-shadow: inset 2px 0 currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)